clang/include/clang/Basic/AttrDocs.td

```diff
@@ -62,6 +62,10 @@ def CodeModelDocs : Documentation {
   let Content = [{
 The ``model`` attribute allows overriding the translation unit's
 code model (specified by ``-mcmodel``) for a specific global variable.
+
+On LoongArch, allowed values are "normal", "medium", "extreme".
```

Comment:
I suspect aaron would like to see more details on this, including what each of those things mean.

Reply:
I've added some description at least on the x86-64 side of things